cmd/svcat/testdata/responses/clusterserviceplans.json

```diff
@@ -44,7 +44,31 @@
         "free": false,
         "clusterServiceClassRef": {
           "name": "4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468"
-        }
+        },
+	"instanceCreateParameterSchema": {
+	  "properties": {
+	    "testInstanceProperty": {
+	      "description": "A test instance property.",
+	      "type": "string"
+	    }
+	  },
+	  "required": [
+	    "testInstanceProperty"
+	  ],
+	  "type": "object"
```

Comment:
I'm unfamiliar with the schema properties. What does the `type` field in this example refer to? `properties`?

Reply:
It's saying that the `parameters` field in the instance/binding request payload is of type `object`, and the `properties` are the key+values expected.